modules/oe_paragraphs_illustrations_lists/config/install/field.field.paragraph.oe_illustration_list_flags.field_oe_subtitle.yml

```diff
@@ -0,0 +1,18 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.paragraph.field_oe_subtitle
+    - paragraphs.paragraphs_type.oe_illustration_list_flags
+id: paragraph.oe_illustration_list_flags.field_oe_subtitle
+field_name: field_oe_subtitle
+entity_type: paragraph
+bundle: oe_illustration_list_flags
+label: Highlight
+description: ''
+required: false
+translatable: true
```

Comment:
We should mark the field storage translatable only.

Reply:
We have the other text fields translatable in this submodule and the rest.